src/main/java/org/mule/service/http/impl/service/HttpMessageLogger.java

```diff
@@ -44,11 +45,15 @@ public void onDataSentEvent(Connection connection, Buffer buffer) {
   }
 
   private void logBuffer(Buffer buffer) {
-    withContextClassLoader(classLoader, () -> {
+    Thread currentThread = currentThread();
```

Comment:
see comment in https://github.com/mulesoft/mule/pull/8711

Reply:
see the answer